s3path.py

```diff
@@ -321,6 +321,8 @@ def rmdir(self, path):
         bucket = resource.Bucket(bucket_name)
         for object_summary in bucket.objects.filter(Prefix=key_name):
             self._boto3_method_with_parameters(object_summary.delete, config=config)
+        if path.is_bucket:
+            bucket.delete()
```

Comment:
In order to pass relevant kwargs and configurations
Please change the delete call like this:

``` python
self._boto3_method_with_parameters(bucket.delete, config=config)
```

Reply:
Done. I added the requested change.